doc/examples/scraper/README.md

```diff
@@ -1,6 +1,6 @@
 # Quick Start Guide: Web Scraper
 In this guide you're going to create a Pachyderm pipeline to scrape web pages. 
-We'll use a standard unix tool, `wget`, to do our scraping. 
+We'll use a standard unix tool, `wget`, to do our scraping.
```

Comment:
Seems reasonable. I think the doc should really inform us how to configure our editors so even when editing english prose we don't end up w noisy diffs like this.